packages/modules/pricing/integration-tests/__tests__/services/pricing-module/calculate-price.spec.ts

```diff
@@ -1134,7 +1134,7 @@ moduleIntegrationTestRunner<IPricingModuleService>({
               {
                 id: "price-set-PLN",
                 is_calculated_price_price_list: true,
-                calculated_amount: 116,
+                calculated_amount: 232,
```

Comment:
any idea what happened here?

Reply:
The way we were sorting the prices was incorrect, see https://medusacorp.slack.com/archives/C05LKT93XNE/p1719482628598669. As I wrote in the PR description, the PR does two things - fixes the sorting (by adding the rule count as the first sorting key), and also enforcing uniqueness on the rules.